.docker/assets/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+npm ci
+echo "npm ci finished"
+
+tail -F /dev/null
```

Comment:
I changed this line to:
```
npm run assets -- --watch
```
and the container stays up and watches the assets. Is there a reason not to do this?

Reply:
Yeah, I thought about that, but it doesn't work because usually the user will want to run `encore dev --watch` for a while and then only at the end do a production build, and the `public/assets/` directory will be filled with all the out of date builds.